.travis.before_install.bash

```diff
@@ -4,10 +4,14 @@ if [ "$TRAVIS_OS_NAME" == "linux" ]; then
   sudo apt-get install enchant -y
 elif [ "$TRAVIS_OS_NAME" == "osx" ]; then
   if [ "$PYTHON" == "/usr/local/bin/python3" ]; then
-    brew install python3
+    # Brewed Python 3.
+    brew upgrade python
```

Comment:
Python 3 is now the default Python on homebrew so `python3` aliases to `python`, but you have to `brew upgrade` to actually get 3 if you already have 2, which is what's in the current Travis VM images.